docs/02_installation/README.md

```diff
@@ -9,11 +9,11 @@ for the remainder of this tutorial.
 
 ## Executive summary
 
-* <a href="#requirements">requirements</a>: Linux, Python 2.7 or 3.5+, environment modules tool (Lmod is recommended)
+* <a href="#requirements">requirements</a>: Linux, Python 2.7 or 3.5+ (with SSL support), environment modules tool (Lmod is recommended)
```

Comment:
the `(with SSL support)` is too much detail here?

Reply:
Speaking as someone who has built python without SSL support, I'm the wrong person to ask. :)

Reply:
I think it is, because the Python used to run EasyBuild usually comes from the OS...